tests/e2e/provisioning/Makefile

```diff
@@ -0,0 +1,12 @@
+APP_NAME = e2e-provisioning-test
+APP_PATH = tests/e2e/provisioning
+BUILDPACK = eu.gcr.io/kyma-project/test-infra/buildpack-golang-toolbox:v20190913-65b55d1
+SCRIPTS_DIR = $(realpath $(shell pwd)/../../..)/scripts
+
+include $(SCRIPTS_DIR)/generic_make_go.mk
+
+VERIFY_IGNORE := /vendor\|/automock\|/testdata
+
+verify:: vet
+
+test-local: ;
```

Comment:
Is it correct?

Reply:
it needs to be override to skip `go test ./...` while calling verify